sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -qU pip awscli boto3 sagemaker transformers==4.9.1
</pre><p>Transformers 4.9.1 is a bit old, they're up to 4.18.0 - does a newer version work? or &gt;= 4.9.1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
updated to newer version of Transformers